common/linux_config.h

```diff
@@ -5,13 +5,13 @@
 #define INITRD_ADDR      (0x20000000)
 #define MACHINE_NUMBER   (0xFFFFFFFF)
 #define ARM9LINUXFW_ADDR (0x08080000)
+#define SOCINFO          (0x10140FFC)
 #define SYNC_ADDR        (0x1FFFFFF8)
 
-#define LINUXIMAGE_FILENAME  "/zImage"
-#define DTB_FILENAME         "/nintendo3ds_ctr.dtb"
-#define ARM9LINUXFW_FILENAME "/arm9linuxfw.bin"
-
-#define FALLBACK_PATH        "/linux"
+#define LINUXIMAGE_FILENAME  "linux/zImage"
```

Comment:
Not related to this PR, but I think we should split this file into 2:

- `linux_config.h`  just containing the Linux and payloads load addresses and filenames.
- `memory.h` or `soc.h` or something like this containing `#defines` for the memory regions/layout